library/payfine/pAY_fINE_cONTROL.java

```diff
@@ -23,8 +23,8 @@ public void setUI(PayFineUI Ui) {               //public void SeT_uI(PayFineUI u
 			throw new RuntimeException("PayFineControl: cannot call setUI except in INITIALISED state");
 		}	
 		this.Ui = Ui;                //uI
-		uI.SeT_StAtE(PayFineUI.uI_sTaTe.READY);
-		StAtE = cOnTrOl_sTaTe.READY;		
+		Ui.setState(PayFineUI.UiState.READY);     //uI.SeT_StAtE(PayFineUI.uI_sTaTe.READY)

```

Comment:
Ui.setState(PayFineUI.UiState.READY) should be ui.setState(PayFineUi.uiState.READY)